### PR TITLE
Fix/inct 178

### PR DESCRIPTION
**Pull request by elem15**
- [x] - mobile menu  (@grippiby)

### CODE REVIEW

**Comment by elem15** [comment]
run build production

**Comment by elem15** [comment]
run build production

**Comment by elem15** [comment]
run build production

**Comment by elem15** [comment]
run build production

**Comment by elem15** [comment]
run build production

**Comment by elem15** [comment]
run build production

File: src/shared/components/dropdown/dropdown.module.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  outline: none;
+}
+
+.dropdownMenuContent {
+  position: relative;
+
+  min-width: 80px;
+  padding: 12px;
+
+  background-color: var(--color-dark-700);
```
Comment (nargiz92) [inline src/shared/components/dropdown/dropdown.module.scss:20]: var(--color-dark-500)
border : 1px solid var(--color-dark-100)

Comment (nargiz92) [reply src/shared/components/dropdown/dropdown.module.scss:20]: border-radius: 2px;

File: src/shared/components/dropdown/dropdown.module.scss [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  &:last-of-type {
+    padding-bottom: 0;
+  }
+
+  &:first-of-type {
+    padding-top: 0;
+  }
+
+  &:not(:last-of-type) {
+    border-bottom: 1px solid var(--color-dark-300);
```
Comment (nargiz92) [inline src/shared/components/dropdown/dropdown.module.scss:70]: none

File: src/shared/components/sidebar/Sidebar.tsx
Comment (nargiz92) [inline src/shared/components/sidebar/Sidebar.tsx:1]: sidebar.module.scss change background border-top: 1px solid var(--dark-300, #333);
background: var(--dark-700, #0D0D0D);

File: src/shared/components/dropdown/dropdown.module.scss
Comment (nargiz92) [inline src/shared/components/dropdown/dropdown.module.scss:1]: component files must begin with a capital letter

File: src/widgets/header/ui/HeaderWidget.tsx [hunk 3]
```diff
@@ -25,19 +39,67 @@ export const HeaderWidget = () => {
...
+    <header className={'header-three sticky-header w-full h-16 sticky lg:relative top-0 z-20'}>
+      <div className="flex justify-between items-center h-16 max-sm:px-6 sm:px-16 py-3 border-b border-dark-300">
+        <Link href="/my-profile" className="text-light-100 text-[26px] font-semibold">
+          Inctagram
+        </Link>
+        <div className="flex justify-center items-center space-x-6">
+          {isAuth && (
+            <div className="hidden lg:flex relative" ref={menuRef}>
+              <NotificationBell toggle={toggle} setToggle={setToggle} />
+              <DropDownNotification toggle={toggle} />
```
Comment (nargiz92) [inline src/widgets/header/ui/HeaderWidget.tsx:51]: component disappears when screen resolution becomes less than 639px

**Review by nargiz92 — APPROVED** [review]

**Pull request merged**